azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachines.java

```diff
@@ -32,20 +32,73 @@
     PagedList<VirtualMachineSize> listSizes(String region) throws CloudException, IOException;
```

Comment:
sounds good, after our discussion i named both of them as "availableSizes()" but "virtualMachines::availableSizesByRegion" seems better, will fix it.